website/src/components/audio-player.tsx

```diff
@@ -3,6 +3,25 @@ import { FiLoader } from "react-icons/fi/index"
 import { MdPauseCircleOutline, MdPlayCircleOutline } from "react-icons/md/index"
 import * as css from "./audio-player.css"
 
+/**
+ * Get our default load status for the audio player.
+ * On some user agents, we need to default to "loaded" because audio can never be loaded in the background.
+ */
+function getDefaultLoadStatus() {
+  // TODO: is there a place to store this instead of doing this check everytime?
+  // happens at this level because vite will run this server-side and crash sometimes
+  // @ts-ignore
+  const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
+  const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
```

Comment:
It looks like audio loading is a problem across browsers on iOS (Tested on Chrome on an iPad using contributed audio from UAT). I know you said safari has some other particular strange behavior as well (mainly re: encoding). 

For load status, would it make sense to just check for iOS here to account for other browsers on iOS devices?

Reply:
I believe all browsers on iOS have to send the safari user agent and are just safari reskins. There might be an exception with people who do "desktop view" which changes the user agent to not be safari. I don't know if that changes this behavior (not loading audio) though and I think it is a kind of a niche scenario.

Reply:
Sounds good to me. Resolving.